docs/user/troubleshooting.md

```diff
@@ -52,7 +52,11 @@ If no pods are shown, etcd will need to be [investigated](#etcd-is-not-running).
 
 ### Unable to SSH into Master Nodes
 
-In order to SSH into the master nodes, it is necessary to include an administrator's SSH key during the installation. If SSH authentication is failing, ensure that the proper SSH key is being used.
+In order to SSH into the master nodes as user `core`, it is necessary to include an administrator's SSH key during the installation. This key can be defined in the `install-config.yml` under the variable `sshKey`. The install-config.yml can be generated by running 
```

Comment:
Mentioning `core` is good.  But I don't think install-config edits are the easiest way to inject the SSH key.  Did you skip the wizard prompt and leave [the associated environment variables][1] unset?

[1]: https://github.com/openshift/installer/blame/v0.5.0/docs/user/environment-variables.md#L37-L41

Reply:
I did not, when it asks you for the ssh key during the install 
? SSH Public Key  [Use arrows to move, type to filter, ? for more help]
  /Users/dobby/.ssh/id_rsa.pub

It actually does not add it in the config ( terraform vars) ass SSH keys. 

Also the env var of OPENSHIFT_INSTALL_SSH_PUB_KEY did not work for me as well. Thus I added it to the config. That worked

Reply:
Tested again and realized the ssh-key was my fault. The installer asks you for a key and shows the path "/Users/dobby/.ssh/id_rsa.pub" to the key. It does not use this path. So if you leave it empty its none. So agree the install-config.yml is not the easiest. Will close pull request and rewrite.